### ISSUE TITLE
Delete Task does not delete tasks

### ISSUE THREAD

**Issue by NVLudwig** [unoplatform/uno.todo#170]
## Current behavior

Pressing the delete task, gives a pressed state but nothing after, and no change is applied to the list

## Expected behavior
Task should be deleted and list updated. If in Task details at the time of deleting it user should navigate back to previous view (false in Desktop mode, If details is in a pane, pane should close). (Bonus points for displaying a Toast reading "Task Deleted"

## How to reproduce it (as minimally and precisely as possible)

Go to any task detail and presse the delete button at the bottom of the screen

## Environment

Mac
OS Chrome Version 100.0.4896.75 (Official Build) (x86_64)
 Private window

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [X] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by agneszitte-nventive** [comment]
@NVLudwig deleting the task was fixed with the changes in the PR I made https://github.com/unoplatform/uno.todo/pull/188
The list is updated and on mobile or narrow desktop version it navigates back.

Issues left to be fixed:
- In Desktop mode, If details are in a pane, the pane should close
- Bonus points for displaying a Toast reading "Task Deleted" (@nickrandolph what do you think regarding this one, and also @NVLudwig there is no toast on the Windows To-Do app no ?)

**Comment by nickrandolph** [comment]
@agneszitte-nventive we don't currently have support in navigation for displaying a toast (but it is a good idea).

**Comment by NVLudwig** [comment]
Yup it's ok,
We don't support toasts so it would have been a nice idea but we aren't in a place to do it.

**Comment by agneszitte-nventive** [comment]
Thanks @NVLudwig for the feedback !
Looking into it as another issue was introduce lately (same as trying to sign out and [deleting a list](https://github.com/unoplatform/uno.todo/issues/252))

**Comment by nickrandolph** [comment]
This PR should address the issue with deleting tasks (and other places message dialog is used): https://github.com/unoplatform/uno.todo/pull/268

**Comment by agneszitte-nventive** [comment]
Deleting task is fixed with https://github.com/unoplatform/uno.todo/pull/268
The last thing to fix @nickrandolph is to close the right pane when when the task is deleted

**Comment by NVLudwig** [comment]
...it deletes every task in the list now...

**Comment by agneszitte-nventive** [comment]
Ok I will double-check that @NVLudwig (cc @nickrandolph)

**Comment by agneszitte-nventive** [comment]
I'm not able to reproduce @NVLudwig issue on my side even with the stating link. 
Waiting on @NVLudwig for him to test on another browser to see if he still can reproduce the issue

**Comment by agneszitte-nventive** [comment]
@nickrandolph heads up that I did some tests and also asked other devs to test on their side and I cannot reproduce @NVLudwig issue. So I did some tests with @NVLudwig and normally it's a cache issue on his side and he wasn't using the latest version.

@nickrandolph the only last thing to fix for this issue is the fact that the task details tab will be closed after deleting the task. So I will assign this issue to you also since you will work on that.
